ChangeLog

```diff
@@ -14,6 +14,7 @@
 - heifsave: improve alpha channel detection [lovell]
 - convi: ensure double sum precision for floats [lovell]
 - improve guard against corrupt ICC profiles with older lcms2 versions [kleisauke]
+- prevent use-after-free during `vips_shutdown` with deprecated plugins [lovell]
```

Comment:
Maybe remove this line, since it's not a use-after-free bug?

Reply:
Yes, I agree, I didn't want this to sound alarmist and yet, reading it back, it definitely sounds alarmist. I've updated the commit message too.